src/backend/mpi/swdsm.cpp

```diff
@@ -1287,6 +1278,6 @@ void printStatistics(){
 void *argo_get_global_base(){return startAddr;}
 size_t argo_get_global_size(){return size_of_all;}
 
-inline unsigned long get_classification_index(uint64_t addr){
+unsigned long get_classification_index(uint64_t addr){
```

Comment:
should this be `static inline` instead? It might be required to move to the header file, and I don't know whether the static is needed then.

Reply:
The issue with this is that `get_classification_index` requires access to `classificationSize` which is only a member of swdsm.cpp. swdsm.h is included in multiple translation units and defining a static `classificationSize` in swdsm.h does not work to solve this problem. For now I am leaving the inlining to the compiler, if you have a better solution feel free to share it otherwise Jenkins should show if inlining carries any importance.